docs/troubleshooting.md

```diff
@@ -9,3 +9,16 @@ To troubleshoot Kangal you will need access to your Kubernetes cluster.
 
 ## Problems with a specific load test
 You can make basic troubleshooting using Kangal API endpoints or either exploring load test Pods if is the case of your backend.
+
+- Get status of the load test 
+```bash
+curl --location --request GET 'http://${KANGAL_PROXY_ADDRESS}/load-test/loadtest-random-name/' 
```

Comment:
is this `--location` param necessary?

Reply:
No, it's a copy-paste leftover, removed.